explore/src/main/scala/explore/targeteditor/AladinCell.scala

```diff
@@ -116,21 +117,17 @@ object AladinCell extends ModelOptics {
             .evalMap(r => gs.setStateAsync(r))
       )
       // Request data again if vizTime changes more than a month
-      .useEffectWithDepsBy((p, _, _, _, _, candidates) => (candidates.awaitOpt, p.obsConf.vizTime))(
-        (props, _, _, _, _, _) => { case (c, _) =>
-          c.value
-            .map(candidatesAwait =>
-              candidatesAwait >>
-                props.ctx.worker.postTransferrable(
-                  CatalogRequest(props.target.get, props.obsConf.vizTime)
-                )
-            )
-            .orEmpty
+      .useEffectWithDepsBy((p, _, _, _, _, candidates) => (candidates, p.obsConf.vizTime))(
```

Comment:
No await?

Reply:
Right, no more await. React will run this effect every time `candidates` change. We only do something when it's in `ReadyNone` state, which means that the fiber has started, but nothing has been received on the stream yet.